test/clustertest/tests/ClusterUpgradeTest.cpp

```diff
@@ -75,13 +79,26 @@ struct ClusterUpgradeTest : tpunit::TestFixture {
             ASSERT_FALSE(system(("cd " + brReleaseDir + " && cd Bedrock && git checkout " + bedrockTagName + "  > /dev/null").c_str()));
 
             // Build the release.
-            ASSERT_FALSE(system(("cd " + brReleaseDir + " && cd Bedrock && make -j8 bedrock  > /dev/null").c_str()));
+            ASSERT_FALSE(system(("cd " + brReleaseDir + " && cd Bedrock && make -j8 > /dev/null").c_str()));
 
             // Save the final product.
+            mkdir(prodBedrockDirName.c_str(), 0755);
             ASSERT_FALSE(system(("mv " + brReleaseDir + "/Bedrock/bedrock " + prodBedrockName).c_str()));
+            ASSERT_FALSE(system(("mv " + brReleaseDir + "/Bedrock/test/clustertest/testplugin/testplugin.so " + prodBedrockPluginName).c_str()));
+
+            // Remove the intermediate dir.
+            rmdir(brReleaseDir.c_str());
+        }
+
+        // Figure out where the new test plugin is.
+        char cwd[1024];
+        if (!getcwd(cwd, sizeof(cwd))) {
+            STHROW("Couldn't get CWD");
         }
+        newTestPlugin = string(cwd) + "/testplugin/testplugin.so";
 
-        tester = new BedrockClusterTester("db,cache,jobs", prodBedrockName);
+        // Load the whole prod cluster with the prod test plugin.
+        tester = new BedrockClusterTester(prodBedrockPluginName, prodBedrockName);
```

Comment:
Noting that db, cache, and jobs are not used in tests, so not a problem that they are being removed here.